src/smb_mission_planner/missions/twist_mission.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, mission_data, reference_frame):
         self.next_twist = False
 
         self.update_rate = rospy.Rate(10)
-        self.twist_pub = rospy.Publisher('/keyboard_teleop/cmd_vel', Twist, queue_size=1)
+        self.twist_pub = rospy.Publisher('/control/keyboard_teleop/cmd_vel', Twist, queue_size=1)
```

Comment:
generally, I would have liked a specific publisher only for the mission planner. But I guess because of the little time left, we can leave it as is.